src/components/RestaurantLookup/RestaurantLookup.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import './RestaurantLookup.css'
+
+class RestaurantLookup extends Component {
+    constructor(props){
+        super(props);
+        this.value = ""
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(event) {
+        this.value = this.capitalize(event.target.value);
```

Comment:
Looks like something that should be done with the component State and `setState`.